content/en/docs/Reference/artifacts.md

```diff
@@ -116,17 +116,21 @@ quay.io/kairos/<flavor>:<flavor_release>-<variant>-<arch>-<device>-<version>-img
 
 ## Framework images
 
-There are 2 framework flavors currently. The `generic` and the `fips` (Defined [here](https://github.com/kairos-io/kairos/blob/5ec84616e898b8079bd92a7424b4c8bf10c5d816/framework-profile.yaml#L9)).
+There are 2 framework flavors currently, `generic` and `fips` (Defined [here](https://github.com/kairos-io/kairos/blob/5ec84616e898b8079bd92a7424b4c8bf10c5d816/framework-profile.yaml#L9)).
```

Comment:
this should actually be removed no? We are dropping that framework profile and the util used to build the image as its now all living in the new repo

Reply:
You're right, will remove